reliability/tasks/CustomizedTask.py

```diff
@@ -0,0 +1,66 @@
+from .GlobalData import global_data
+from tasks.utils.oc import oc
+import os
+import logging
+
+class OCTask():
+    def __init__(self, command, kubeconfig):
+        self.command = command
+        self.kubeconfig = kubeconfig
+        
+    def execute(self):
+        return oc(self.command.lstrip("oc "), self.kubeconfig)
```

Comment:
This is fine for now, but a future enhancement could be to run arbitrary external commands.   This might be useful for monitoring system resources or triggering actions outside of the constraints of "oc"

Reply:
Hi Mike, Thanks for the comment. I will consider this in future enhancement.